CHANGELOG.unreleased.md

```diff
@@ -38,6 +39,7 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- xo-server-sdn-controller patch
```

Comment:
Really?

Reply:
Yes it's a bug fix.

Reply:
No, this PR does not contain any change on this plugin, no need to release it.

Reply:
Oh right it's only on xo-web and the plugin is changed in the other PR ^^'